ext/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+add_subdirectory(fonts)
```

Comment:
As external components should be moved out-of-tree it is maybe better to introduce a dedicated fonts repo.

Reply:
The font must live in `ext/` because of its license.  I don't believe it's going to be forced out-of-tree.

Reply:
> The font must live in ext/ because of its license. 

Is this not one of the reason to move suche modules out-of-tree (@nashif)?

Reply:
As I've removed `display_console.c` I've also eliminated the font.